src/cohortManager.js

```diff
@@ -0,0 +1,89 @@
+const Cohort = require('../src/cohort')
+const Student = require('../src/students')
+
+class CohortManager {
```

Comment:
Nice and clear layout and naming of your methods! made it easy to navigate your code, good job!